### PR TITLE
Feature/first version

### PR DESCRIPTION
**Pull request by dennis-montana**
This is the first version of the Sentry plugin to log crashes and send them to Sentry.

### CODE REVIEW

File: src/ng-sentry.common.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        if (this.breadcrumbs.length > this.maxAmountOfBreadcrumbs) {
+            this.breadcrumbs.shift();
+        }
+    }
+
+    public saveCrash(errorMessage: string, errorDetails: string): void {
+        this.saveBreadcrumb('' + errorDetails, 'crash start');
+
+        const crashData = {
+            project: '253147',
```
Comment (corne-de-bruin) [inline src/ng-sentry.common.ts]: Should this be hardcoded?

File: src/ng-sentry.common.ts [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+            },
+            breadcrumbs: {
+                values: this.breadcrumbs
+            },
+            environment: this.environment,
+            release: this.versionName,
+            contexts: {
+                device: {
+                    family: platform.device.manufacturer,
```
Comment (corne-de-bruin) [inline src/ng-sentry.common.ts]: Is platform.device always available, if not this way the plugin itself will crash?

File: src/ng-sentry.common.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        }
+
+        const crashes = JSON.parse(getString(this.storageKey));
+        for (let crash of crashes) {
+            this.submitCrash(crash);
+        }
+    }
+
+    private submitCrash(crash) {
+        console.log('submitCrash', crash.message);
```
Comment (corne-de-bruin) [inline src/ng-sentry.common.ts]: Do you want to keep this logging statement in final version?

**Review by corne-de-bruin — CHANGES_REQUESTED** [review]

File: src/ng-sentry.common.ts (hunk 2 above)
Comment (dennis-montana) [reply src/ng-sentry.common.ts]: According to the documentation (https://docs.nativescript.org/cookbook/platform) the properties can be accessed without a check if they're available.

**Review by corne-de-bruin — APPROVED** [review]

**Pull request merged**